packages/fxa-content-server/app/tests/spec/lib/email-domain-validator.js

```diff
@@ -189,32 +192,34 @@ describe('lib/email-domain-validator', () => {
     return checkEmailDomain($el, view)
       .then(() => assert.fail('The promise should have rejected'))
       .catch(err => {
-        assert.equal(err, REJECTION_CODE);
-        assert.isTrue(
-          AuthErrors.toInvalidEmailDomainError.calledOnceWith(
-            'no.records.co.gd'
-          )
-        );
-        assert.isTrue(
-          view.showValidationError.calledOnceWith(
-            $el,
-            AuthErrors.toInvalidEmailDomainError.returnValues[0]
-          )
-        );
-
-        assert.isTrue(
-          view.logEvent.calledWith('emailDomainValidation.triggered')
-        );
-        assert.isTrue(view.logEvent.calledWith('emailDomainValidation.fail'));
-        assert.isTrue(
-          view.logFlowEvent.calledWith('email-domain-validation.triggered')
-        );
-        assert.isTrue(
-          view.logFlowEvent.calledWith('email-domain-validation.block')
-        );
-
-        AuthErrors.toInvalidEmailDomainError.restore();
-        xhr.ajax.restore();
+        p.delay(50).then(() => {
```

Comment:
Since this arrow function uses a `{`, You'll need to add `return` here or else any errors in this test will not be caught correctly.

Reply:
Good catch.  I'll remove the `{`.  Thanks!